org.eclipse.jdt.ls.product/pom.xml

```diff
@@ -113,6 +128,21 @@
 							</resources>
 						</configuration>
 					</execution>
+					<execution>
+						<id>copy-config-linux-arm</id>
+						<phase>package</phase>
+						<goals>
+							<goal>copy-resources</goal>
+						</goals>
+						<configuration>
+							<outputDirectory>${project.build.directory}/repository/config_linux_arm</outputDirectory>
+							<resources>
+								<resource>
+									<directory>${basedir}/target/products/languageServer.product/linux/gtk/aarch64/configuration</directory>
+								</resource>
+							</resources>
+						</configuration>
+					</execution>
```

Comment:
Just one minor thing here. If you look lower in the file, you'll see executions for `copy-config-${os}-${arch}-slim`. These are for the syntax server. It is a slimmed down version of the language server that doesn't import projects, and as a result has fewer features, but can serve users while the full-featured (standard) server completes importing the project. In VS Code we have a hybrid mode where the syntax server starts up first, and is replaced (shut down) by the standard server.

Adding copy-config-mac-arm-slim, copy-config-linux-arm-slim would ensure the lighter weight language server is available for those architectures as well.